collections/guides/tokenize_customer_data_in_browser/index.md

```diff
@@ -25,7 +25,7 @@ Basis Theory is built to handle any payload of data you can serialize. The follo
 
 <span class="base-alert warning">
   <span>
-    You'll need a <code>Client-side</code> [Application](https://docs.basistheory.com/api-reference/#applications) created with the <code>token:create</code> permission for this step.
+    You'll need a <code>Client-side</code> [Application](https://docs.basistheory.com/api-reference/#applications) created with the <code>token:pii:create</code> permission for this step.

```

Comment:
We also need to update the `submit()` handler example to override default Privacy Settings for `pii` classification:
```
  const token = await bt.tokens.create({
    type: 'token',
    data: {
      name,
      ssn,
      phone
    },
    privacy: {
      classification: 'pii'
    });
```